src/paymentservice/sentry.js

```diff
@@ -1,14 +1,18 @@
 const Sentry = require("@sentry/node");
+require("@sentry/tracing");
+const { ProfilingIntegration } = require("@sentry/profiling-node");
```

Comment:
do we also have to add to the payment service dockerfile?

Reply:
Oh yes, I thought we only run on the frontend, let me fix that

Reply:
I should have checked before speaking, paymentservice already installs the exact same set of utils